docs/waf-rules.md

```diff
@@ -0,0 +1,84 @@
+---
+
+template:      article
+reviewed:      2022-03-01
+naviTitle:     WAF rules
+title:         Web Application Firewall
+excerpt:       Protect your App against common attacks and harmful requests
+group:         platform
+stack:         all
+dontList:      true
+dontIndex:     true
+---
+
+
+## Problem
+
+Your App is exposed to the World Wide Web to give your users access. Unfortunately not every request is made by friendly humans. In fact there are bots, crawlers and attackers that scan your site for vulnerabilities. Even if they are not successful those requests are harmful because they eat resources which are meant for your users.
+
+## Solution
+
+Instead of a full-fledged Web Application Firewall we block most common automated request patterns.
+
+## Basic WAF rules
+
+By default "Basic WAF rules" are enabled, you can disable them in the dashboard. 
+
+### Blocked files
+
+* common dotfiles
```

Comment:
Just my ignorance I guess, but I didn't see the dotfiles blocked in the gof PR. Just curious where that happens.